### PR TITLE
Pass environment variables to executor

### PR DESCRIPTION
**Pull request by sandyjuventus**
* Add feature to allow executor to fetch logger binary
* Add environment variables to executor

### CODE REVIEW

File: cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/ExecutorConfig.java [hunk 1]
```diff
@@ -303,6 +303,9 @@ public String getVolumeDriver() {
         uris.add(executorLocation.toString());
         uris.add(cassandraLocation.toString());
         uris.add(jreLocation.toString());
+        if(System.getenv().containsKey("LOGGER_LOCATION")) {
```
Comment (aaronjwood) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/ExecutorConfig.java]: If it's easier, you could add a default for this in here https://github.com/verizonlabs/dcos-cassandra-service/blob/master/cassandra-executor/src/dist/conf/executor.yml so you don't have to worry about it never being undefined.

Comment (sandyjuventus) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/ExecutorConfig.java]: Don't you mean scheduler.yml? Also, the default values mentioned right are not reachable from our datacenter.

Comment (aaronjwood) [inline cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/ExecutorConfig.java]: Yup, my fault. I was looking at the file name and saw that it was for the executor. Forgot that this part is used in the scheduler.

**Comment by aaronjwood** [comment]
@sandyjuventus looks like this broke something, the scheduler can't deploy now. stderr shows
```
./scheduler/conf/scheduler.yml has an error:
  * Failed to parse configuration at: executor; Instantiation of [simple type, class com.mesosphere.dcos.cassandra.common.config.ExecutorConfig] value failed (java.lang.NullPointerException): null
 at [Source: N/A; line: -1, column: -1] (through reference chain: com.mesosphere.dcos.cassandra.common.config.MutableSchedulerConfiguration["executor"])
```

**Pull request merged**